### ISSUE TITLE
Oh no, is it dead?

### ISSUE THREAD

**Issue by yakimant** [clarkgrubb/hyperpolyglot#101]

**Comment by limeb** [comment]
Its down for me also.  It is one of my favorite sites.

**Comment by waldyrious** [comment]
@clarkgrubb, how about having the site hosted in GitHub Pages (with a custom domain, so that externally it still works the same, via hyperpolyglot.org)? That way even if the domain has issues, the site would still be accessible via https://hyperpolyglot.github.io.

By the way, it looks like maintenance has been an issue too, given the [pending PRs](https://github.com/clarkgrubb/hyperpolyglot/pulls). Perhaps it could be a good idea to create a GitHub organization and give membership to some of the [contributors](https://github.com/clarkgrubb/hyperpolyglot/graphs/contributors), to distribute maintenance duties and prevent issues and PRs from going unaddressed for a long time.

**Comment by limeb** [comment]
I agree with the github pages.  The maintenance part I'm not really sure.  I'm on the fence for that probably.

**Comment by waldyrious** [comment]
@clarkgrubb any thoughts on the project maintenance suggestions? I'd be happy to volunteer some time to help out, of course following any guidelines you may find reasonable.
